spec/rails_helper.rb

```diff
@@ -0,0 +1,64 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require_relative '../config/environment'
+# Prevent database truncation if the environment is production
+abort('The Rails environment is running in production mode!') if Rails.env.production?
+require 'rspec/rails'
```

Comment:
- [ ] I have to add the comment here as you do not have any spec file to make the comment, According to project requirement, you are to create `unit tests for User, Post, Comment, and Like model`, I currently do not see any file for your test, kindly write your unit test and submit again